guides/doc-Upgrading_and_Updating/topics/upgrading_overview.adoc

```diff
@@ -57,41 +43,61 @@ If you customize configuration files, manually or use a tool such as Hiera, thes
+For more information, see the https://access.redhat.com/documentation/en-us/red_hat_satellite/{ProjectVersionPrevious}/html/upgrading_and_updating_red_hat_satellite/[_{UpgradingDocTitle} to {ProjectVersionPrevious}_].
 endif::[]
 
 ifndef::satellite[]
 You can upgrade to {ProjectName} {ProjectVersion} from {ProjectName} {ProjectVersionPrevious}.
+
 {ProjectServer}s and {SmartProxyServer}s on earlier versions must first be upgraded to {Project} {ProjectVersionPrevious}.
 For more information, see the {Project} {ProjectVersionPrevious} Upgrade documentation.
 endif::[]
 
-.Overview of {Project} {ProjectVersion} Upgrade Paths
-ifdef::satellite[]
-image::satellite_6.4_upgrade_paths.png[Overview of {Project} {ProjectVersion} Upgrade Paths]
+=== High-Level Upgrade Steps
 
-WARNING: Upgrading from the Beta to GA version is not supported.
-endif::[]
-
-The high level steps in upgrading to {Project} {ProjectVersion} are as follows.
+The high-level steps in upgrading {Project} to {ProjectVersion} are as follows.
 
 ifdef::satellite[]
 . Clone your existing {ProjectServer}s. For more information, see xref:cloning_satellite_server[].
 endif::[]
 . Upgrade {ProjectServer} to {ProjectVersion}.
 For more information, see xref:Upgrading_Server_{context}[].
-. Upgrade all {SmartProxyServer}s to {ProjectVersion}. For more information, see xref:upgrading_capsule_server[].
+. Upgrade all {SmartProxyServer}s to {ProjectVersion}.
+For more information, see xref:upgrading_capsule_server[].
 ifdef::katello,orcharhino,satellite[]
-. Upgrade to {project-client-name} on all {Project} clients.
-For more information, see xref:upgrading_clients[].
+. Upgrade to {project-client-name} on all content hosts.
+For more information, see xref:upgrading_content_hosts[].
+endif::[]
+ifdef::satellite[]
+. Optional: After you upgrade your {Project}, you can also upgrade the operating system on your {ProjectServer}s and {SmartProxies} to {RHEL} 8.
+endif::[]
+ifndef::satellite,foreman-deb,orcharhino[]
+. Optional: After you upgrade your {Project}, you can also upgrade the operating system on your {ProjectServer}s and {SmartProxies} to Enterprise Linux 8.
+endif::[]
+ifdef::foreman-el,katello,satellite[]
+There are two ways of upgrading your OS:
+
+* xref:upgrading-project-in-place-using-leapp_{context}[]
+* xref:migrating-project-to-a-new-el-system_{context}[]
+endif::[]
+
+////
+ifdef::satellite[]
+During an upgrade of {ProjectServer}, you must observe the correct upgrade path depending on your network environment:
+
+.Overview of {ProjectServer} Upgrade Paths in Connected and Disconnected Network Environments
+image::satellite_6.4_upgrade_paths.png[Overview of {ProjectServer} Upgrade Paths in Connected and Disconnected Network Environments]
 endif::[]
+////
+
+include::../../common/modules/ref_post-upgrade.adoc[leveloffset=+2]
```

Comment:
Is this a comment block? https://docs.asciidoctor.org/asciidoc/latest/comments/#comment-blocks

If so, maybe remove it before merging.